### PR TITLE
Deploy to Production sequentially when pre-deployment checks are defined

### PR DESCRIPTION
**Pull request by take-five**
When pre-deployment checks require some user input (that's what they
were designed for), deploying in parallel makes things cumbersome:
1. User must abort ALL parallel branches, aborting just one doesn't have
   the desired effect.
2. Request for input from multiple branches may lost in the old UI,

For now the solution is to simply force sequential mode if
pre-deployment checks are defined. In the future we can develop some API
for that.

### CODE REVIEW

File: src/com/salemove/Deployer.groovy [hunk 1]
```diff
@@ -68,7 +68,7 @@ class Deployer implements Serializable {
     def defaultArgs = [
       kubernetesNamespace: 'default',
       lockGlobally: true,
-      deploymentUpdateTimeout: [time: 10, unit: 'MINUTES']
+      deploymentUpdateTimeout: [time: 10, unit: 'MINUTES'],
```
Comment (deiwin) [inline src/com/salemove/Deployer.groovy:71]: Unnecessary change.

**Comment by take-five** [comment]
!deploy

**Comment by take-five** [comment]
!check

**Comment by sm-deployer** [comment]
Deploy failed or was aborted. @take-five, please check the logs [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/4//console)).

**Comment by take-five** [comment]
!check

**Comment by take-five** [comment]
!deploy

**Comment by sm-deployer** [comment]
@take-five, the changes were validated in acceptance. Please click **Proceed** [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/5/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/5//console)) to continue the deployment.

**Comment by sm-deployer** [comment]
Deploy failed or was aborted. @take-five, please check the logs [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/5/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/5//console)).

**Comment by take-five** [comment]
!deploy

**Comment by take-five** [comment]
!deploy

**Comment by sm-deployer** [comment]
@take-five, the changes were validated in acceptance. Please click **Proceed** [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/6/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-109/6//console)) to continue the deployment.
